src/dsp_tools/models/permission.py

```diff
@@ -22,26 +22,6 @@ def __str__(self):
         return tmp[self.value]
 
 
-class PermissionsIterator:
-    _permissions: "Permissions"
-    _group: list[str]
-    _index: int
-
-    def __init__(self, permissions: "Permissions"):
-        self._permissions = permissions
-        self._index = 0
-
-    def __next__(self):
-        if len(self._permissions.permissions) == 0 and self._index == 0:
-            return None, None
-        elif self._index < len(self._permissions.permissions):
-            tmp = self._prefixes[self._index]
```

Comment:
If I were to guess, the intention of this code was to make `Permissions` iterable... but I'm not even sure this would have worked like that. And if it's not used, it should definitely be removed.